actions/code.js

```diff
@@ -17,21 +17,32 @@ function wait(timeout) {
 
 // eslint-disable-next-line consistent-return,func-names
 exports.process = async function (msg, conf, snapshot) {
+  const vmExports = {};
   const ctx = vm.createContext({
-    _,
+    // Node Globals
+    Buffer,
+    clearInterval,
+    clearTimeout,
     console,
+    exports: vmExports,
+    global: {},
+    module: { exports: vmExports },
```

Comment:
what is the use-case for this? The component won't be using this exports anyway, right?

Reply:
Most likely not.  However:
* we want to avoid any crashes when people assign values to exports.
* the legacy behavoir is to support exports.